### PR TITLE
Update "Getting Involved" and "Next Steps" with discord, DAG, Crosslink overview, and simtfl overview.

### LINKED ISSUE #91: Update "Getting Involved" and "Next Steps" with discord, DAG, Crosslink working doc, and simtfl links.
### Suggested Improvement

I just [posted a bunch of status/resource updates to the `Proof-of-Stake` channel on Zcash R&D discord](https://discord.com/channels/809218587167293450/826162958027063377/1164593547471765516), and all of that information needs to be consolidated into the book so we have a single URL people can see to glean all that info.

Here's a paste of the contents to feed into a PR:

> Hi folks, as mentioned on today's http://zcasharborist.org/ call, at ECC we're migrating all of our discussion about "Trailing Finality Layer" to this channel to ensure its development is open to any collaborators and to enable people to follow along closely if they'd like.
> For background on "Trailing Finality Layer" ( aka TFL)  our initial announcement of that approach was https://electriccoin.co/blog/the-trailing-finality-layer-a-stepping-stone-to-proof-of-stake-in-zcash/ . For the current state of the project, we have in-progress semi-inconsistent high level description here: https://electric-coin-company.github.io/tfl-book/
> A big piece that's missing from the TFL book is that we've improved upon the "Snap-and-Chat" construction in the [Ebb-and-Flow Protocols](https://eprint.iacr.org/2020/1091.pdf) paper. That construction has a big flaw (or if not a direct flaw at least very important missing detail) about how to ensure transactions remain valid/coherent during "sanitization". We're developing an improvement that still achieves the Ebb-and-Flow properties while being safe/coherent for transaction semantics called Crosslink (largely thanks to @Daira Emma !).
> So big picture next steps are to update the book contents to introduce Crosslink and describe the status of Crosslink R&D. As for Crosslink itself, the main remaining piece is to finish a security argument for Safety. Meanwhile, we're also starting on a simulator we can use to analyze security (such as simulating specific attacks). That project is here https://github.com/zcash/simtfl and is still a prototype that does not yet simulate crosslink or Zcash NU5.
> The current draft documentation for Crosslink is [here](https://hackmd.io/JqENg--qSmyqRt_RqY7Whw?view).

### PR DESCRIPTION
**Pull request by nathan-at-least**
This closes #91 and also updates the "Status and Next Steps" page substantially.

This directly extends #87.

### CODE REVIEW

File: src/introduction/status-and-next-steps.md [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
 
-This design is at a very early stage and lacks substantial clarity around essential details. These details must be clarified before this proposal would be ready for the [Zcash Improvement Proposals](https://zips.z.cash) process. They include:
+### This Book
 
-- PoS protocol selection,
+This book is intended to become both a high level overview and introduction to [TFL](../terminology.md#definition-tfl) as well as a full specification.
+
+### Crosslink
+
+The current heart of the design work is an in-progress hybrid consensus protocol construction called [Crosslink](../terminology.md#definition-crosslink). This is an in-development implementation of the [TFL](../terminology.md#definition-tfl) design goals. The essential design details are specified, along with a security argument of its [liveness](../terminology.md#definition-liveness). The security argument for its [safety](../terminology.md#definition-liveness) is in progress.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:13]:
```suggestion
The current heart of the design work is an in-progress hybrid consensus protocol construction called [Crosslink](../terminology.md#definition-crosslink). This is an in-development realization of the [TFL](../terminology.md#definition-tfl) design goals. The essential design details are specified, along with a security argument for its [liveness](../terminology.md#definition-liveness). The security argument for its [safety](../terminology.md#definition-liveness) is in progress.
```

File: src/introduction/status-and-next-steps.md [hunk 2]
```diff
@@ -1,29 +1,49 @@
...
+
+The draft specification and security arguments of Crosslink currently live on [this hackmd](https://hackmd.io/JqENg--qSmyqRt_RqY7Whw?view).
+
+### `simtfl`
+
+We've begun creating a simulator called [`simtfl`](../terminology.md#definition-tfl) which we will use to model security and abstract performance concerns. Its development is tracked at <https://github.com/zcash/simtfl>.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:19]:
```suggestion
We've begun creating a simulator called [`simtfl`](../terminology.md#definition-simtfl) which we will use to model security and abstract performance concerns. Its development is tracked at <https://github.com/zcash/simtfl>.
```

File: src/introduction/status-and-next-steps.md [hunk 3]
```diff
@@ -1,29 +1,49 @@
...
+
+- PoS subprotocol selection,
 - Issuance and supply mechanics, such as how much ZEC stakers may earn,
+- Integrated Zcash transaction semantics,
 - A transition plan from current Zcash mainnet to this protocol design,
-- The specifics of how PoW and PoS safely integrate,
-- Security and safety analysis,
-- Economic analysis,
-- and more.
+- [ZIP](../terminology.md#definition-tfl)s specifying the above to the level of specificity required by ZIPs,
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:27]:
```suggestion
- [ZIP](../terminology.md#definition-zip)s specifying the above to the level of specificity required by ZIPs,
```

File: src/introduction/status-and-next-steps.md [hunk 4]
```diff
@@ -1,29 +1,49 @@
...
+- Security and safety analyses,
+- Economic analyses
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:29]:
```suggestion
- Economic analyses.
```

File: src/introduction/status-and-next-steps.md [hunk 5]
```diff
@@ -1,29 +1,49 @@
...
+
+This list may be incomplete and as the design matures the need for major new components may be revealed.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:31]:
```suggestion
This list may be incomplete, and as the design matures the need for major new components may be revealed.
```

File: src/introduction/status-and-next-steps.md [hunk 6]
```diff
@@ -1,29 +1,49 @@
...
 
 ## Next Steps
 
 This design proposal is being developed by [ElecticCoin Co](https://electriccoin.co/) as the first major milestone in our focus of deploying Proof-of-Stake to the Zcash protocol. Our rough near term plan for this proposal is as follows:
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:35]:
```suggestion
This design proposal is being developed by [Electric Coin Company](https://electriccoin.co/) as the first major milestone in our focus of deploying Proof-of-Stake to the Zcash protocol. Our rough near-term plan for this proposal is as follows:
```

File: src/introduction/status-and-next-steps.md [hunk 7]
```diff
@@ -1,29 +1,49 @@
...
 
-1. Write a very high-level design overview which lacks many details but clarifies the general approach.
-2. Get early feedback from Zcash community and consensus protocol experts on this high level overview.
-3. If there are flaws so substantial that we decide the approach is infeasible, start over with a different PoS transition design.
-4. Otherwise, refine the high-level "overview" into a concrete, comprehensive proposal through multiple milestones with wide review.
-5. As the concrete proposal approaches maturity, draft one or more [Zcash Improvement Proposals](https://zips.z.cash).
-6. Follow the general Zcash process for proposal/review/refinement.
+1. Complete [Crosslink](../terminology.md#definition-crosslink) protocol specification.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:37]:
```suggestion
1. Complete the [Crosslink](../terminology.md#definition-crosslink) protocol specification.
```

File: src/introduction/status-and-next-steps.md [hunk 8]
```diff
@@ -1,29 +1,49 @@
...
+2. Complete core security arguments for [Crosslink](../terminology.md#definition-crosslink).
+3. Define the [Major Missing Components](#major-missing-components) above, including considerations such as issuance mechanics and Proof-of-Stake mechanisms.
+4. Complete auxillary security arguments and analyses, such as specific attack scenarios, game-theoretic security, and so forth.
+5. Mature [`simtfl`](../terminology.md#definition-tfl) to analyze all cases of interest.
+6. Follow the general Zcash process for proposal/review/refinement, including proposing  or more [ZIP](../terminology.md#definition-tfl)s.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:42]:
```suggestion
6. Follow the general Zcash process for proposal/review/refinement, including proposing one or more [ZIP](../terminology.md#definition-tfl)s.
```

File: src/introduction/status-and-next-steps.md [hunk 9]
```diff
@@ -1,29 +1,49 @@
...
-2. Get early feedback from Zcash community and consensus protocol experts on this high level overview.
-3. If there are flaws so substantial that we decide the approach is infeasible, start over with a different PoS transition design.
-4. Otherwise, refine the high-level "overview" into a concrete, comprehensive proposal through multiple milestones with wide review.
-5. As the concrete proposal approaches maturity, draft one or more [Zcash Improvement Proposals](https://zips.z.cash).
-6. Follow the general Zcash process for proposal/review/refinement.
+1. Complete [Crosslink](../terminology.md#definition-crosslink) protocol specification.
+2. Complete core security arguments for [Crosslink](../terminology.md#definition-crosslink).
+3. Define the [Major Missing Components](#major-missing-components) above, including considerations such as issuance mechanics and Proof-of-Stake mechanisms.
+4. Complete auxillary security arguments and analyses, such as specific attack scenarios, game-theoretic security, and so forth.
+5. Mature [`simtfl`](../terminology.md#definition-tfl) to analyze all cases of interest.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:41]:
```suggestion
5. Mature [`simtfl`](../terminology.md#definition-simtfl) to analyze all cases of interest.
```

File: src/introduction/status-and-next-steps.md [hunk 10]
```diff
@@ -1,29 +1,49 @@
...
+6. Follow the general Zcash process for proposal/review/refinement, including proposing  or more [ZIP](../terminology.md#definition-tfl)s.
 7. Follow the general Zcash governance process for proposal acceptance.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:43]:
```suggestion
7. Follow the general Zcash governance process for proposal review and refinement.
```

File: src/introduction/status-and-next-steps.md [hunk 11]
```diff
@@ -1,29 +1,49 @@
...
 8. If accepted, productionize the proposal in ECC products and collaborate with other implementors who implement the proposal.
 9. Celebrate when the proposal is activated on Mainnet. 🎉
+
+The fine-grained day-to-day goals and tasks for this project are present in [the Zcash Developers Hub](https://zcash.github.io/developers) in the [TFL-focused DAG](https://zcash.github.io/developers/zcash-tfl-dag).
+
+Please also see [Getting Involved](./getting-involved.md) if you are interested in tracking this progress more closely or contributing.
```
Comment (daira) [inline src/introduction/status-and-next-steps.md:49]:
```suggestion
Please also see [Getting Involved](./getting-involved.md) if you are interested in tracking this progress more closely, or in contributing.
```

File: src/introduction/status-and-next-steps.md [hunk 12]
```diff
@@ -1,29 +1,49 @@
...
+This is an early and incomplete protocol design proposal. It has not been well vetted for feasibility and safety. It has not had broad review from the Zcash community, so its status on any Zcash roadmap is undetermined.
 
-## Major Missing Elements
+## Current Components
 
-This design is at a very early stage and lacks substantial clarity around essential details. These details must be clarified before this proposal would be ready for the [Zcash Improvement Proposals](https://zips.z.cash) process. They include:
+### This Book
 
-- PoS protocol selection,
+This book is intended to become both a high level overview and introduction to [TFL](../terminology.md#definition-tfl) as well as a full specification.
```
Comment (nathan-at-least) [inline src/introduction/status-and-next-steps.md]:
```suggestion
This book is intended to become a high level overview and introduction to [TFL](../terminology.md#definition-tfl) as well as a full specification.
```

**Review by nathan-at-least — COMMENTED** [review]
Applied one of Daira's suggestions.

File: src/terminology.md [hunk 13]
```diff
@@ -6,28 +6,80 @@ Definitions are sorted alphabetically.
...
+
+
+
+
+
+
+
+
+
+</pre>
```
Comment (nathan-at-least) [inline src/terminology.md:85]: I don't understand this diff hunk. Does it force appending a bunch of whitespace? Why?

**Review by nathan-at-least — COMMENTED** [review]
All changes look good _except_ for the mysterious `<pre>` whitespace block in `src/terminology.md`.

**Pull request merged**